vpn.sh

```diff
@@ -1,12 +1,13 @@
 #!/usr/bin/env sh
 
-MAX_RETRY_COUNT=5
-MAX_SETUP_ATTEMPTS=3
-WAIT_KVN_TIMEOUT=20
 VPN_CLIENT_VERSION=$1
 VPN_USERNAME=$2
 VPN_PASSWORD=$3
 VPN_AUTH_CODE=$4
+MAX_CURL_ATTEMPT=$5
+MAX_SETUP_ATTEMPTS=$6
+WAIT_KVN_TIMEOUT=$7
+
 DESIRED_MTU=${5:-1500}  # Default to 1500 if not set
```

Comment:
please be careful with those, as this PR broke default MTU settings. Fix was allied at https://github.com/trilogy-group/action-kerio-vpn/commit/c504598cd817703292c622607fdd9c800fdf52d7